packages/lodestar/src/network/options.ts

```diff
@@ -22,5 +23,13 @@ export const defaultNetworkOptions: INetworkOptions = {
   discv5FirstQueryDelayMs: 1000,
   localMultiaddrs: ["/ip4/0.0.0.0/tcp/9000"],
   bootMultiaddrs: [],
+  // rate tracker options per 1 minute
+  // per peer per minute, allow to serve up to 5 requests and 500 blocks
+  // total: make 4x peer params
+  requestCountTotalLimit: 200,
+  requestCountPeerLimit: 50,
+  blockCountTotalLimit: 2000,
+  blockCountPeerLimit: 500,
+  rateTrackerTimeoutMs: 60 * 1000,
```

Comment:
If dependency loops allow consider moving to rate limit file so that all the logic and reasoning is in one place